packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts

```diff
@@ -117,13 +135,33 @@ export const AutocratProposalIndexer: IntervalFetchIndexer = {
         for (const token of [basePass, baseFail, quotePass, quoteFail]) {
           const metadata = await enrichTokenMetadata(token, provider);
           const storedMint = await getMint(provider.connection, token);
+          // NOTE: THIS IS ONLY FOR PROPOSALS AND ONLY FOR BASE / QUOTE CONDITIONAL
+          // TODO: Need to update this for the symbol, name, and image_url base on pass / fail
+          const isQuote = [quoteFail, quotePass].includes(token);
+          const isFail = [quoteFail, baseFail].includes(token);
+          // TODO: Need to get metadata for the base token
+          // TODO: This needs to be refactored
+          let noMetadataName = isFail ? `Proposal ${proposal.account.number}: f${baseTokenMetadata.symbol}` : `Proposal ${proposal.account.number}: p${baseTokenMetadata.symbol}`;
+          let noMetadataSymbol = isFail ? `f${baseTokenMetadata.symbol}` : `p${baseTokenMetadata.symbol}`;
+          let imageUrl;
+          if(isQuote){
+            noMetadataName = isFail ? `Proposal ${proposal.account.number}: fUSDC` : `Proposal ${proposal.account.number}: pUSDC`;
+            noMetadataSymbol = isFail ? `fUSDC` : `pUSDC`;
+          }
```

Comment:
this is kinda weird branching logic, I'd do
`let imageUrl, defaultMetadataName, defaultMetadataSymbol;`
and then follow that with an if else on `isQuote`

Reply:
or, if you want to be clever:
```
let passOrFailPrefix = isFail ? "f" : "q";
let baseSymbol = isQuote ? "USDC" : baseTokenMetadata.symbol;
let defaultSymbol = passOrFailPrefix + baseSymbol;
let defaultName = `Proposal ${proposal.account.number}: ${defaultSymbol}`